addons/storyshots/stories/required_with_context/Button.stories.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from './Button';
+import Button from '@storybook/components/dist/demo/Button';
```

Comment:
My understanding is that it's generally not a great idea to ask users to require from files inside your package. It's obviously not universal not to do it (e.g. `react-test-renderer`), but I think you'll notice most packages are moving away from it. The main reason is it is brittle:

1. You lose the flexibility to re-arrange how your files are organised in your package--certain files have to remain no matter what.

2. You expose details of your build process (cf `/dist/` being in the path). This is restrictive again--e.g. what happens in the future when we don't need babel any more and `dist` is redundant? Or if `lerna` has some great new technique that favours a different naming of output file?

3. Also, tooling understands the module entry point (`package.json#main`), whereas other entry points are just not going to be picked up. This also means new devs don't have any way to know which of our files users may be importing, exacerbating #1.

It doesn't seem that hard to export everything from a common entry file to me.

```js
import DemoButton from '@storybook/components';
```

OTOH, Storybook already asks users to import from paths a bit so if you want to do it, fine.